packages/react-native-worklets/Common/cpp/worklets/WorkletRuntime/RuntimeManager.cpp

```diff
@@ -0,0 +1,107 @@
+#include <worklets/NativeModules/JSIWorkletsModuleProxy.h>
+#include <worklets/WorkletRuntime/RuntimeManager.h>
+
+#include <utility>
+
+namespace worklets {
+
+const std::string uiRuntimeName{"UI"};
```

Comment:
Do we really need to have both runtime ID and runtime name? Also, it looks like runtime names should be unique (as they are used as map keys) so it seems they serve the same purpose as runtime IDs.